libs/blocks/graybox/graybox.css

```diff
@@ -0,0 +1,283 @@
+  top: 0;
+  width: 100%;
+  z-index: 1000;
+}
+
+body.gb-overlay::before {
+  position: fixed;
+}
+
+/* The elements that should appear above the overlay */
+.gb-changed {
+  position: relative;
+  z-index: 1001;
+}
+
+.graybox-container {
+  position: fixed;
+  right: 0;
+  top: 15%;
+  z-index: 2001;
+}
+
+.graybox-container .gb-toggle {
+  background-color: var(--gb-outline-color);
+  border: var(--gb-container-border);
+  cursor: pointer;
+  height: 64px;
+  left: -40px;
+  position: absolute;
+  width: 40px;
+}
+
+.graybox-container .gb-toggle::before {
+  border-bottom: var(--gb-toggle-arrow);
+  border-left: var(--gb-toggle-arrow);
+  border-radius: 2px;
+  content: " ";
+  cursor: pointer;
+  display: block;
+  height: 8px;
+  margin-left: 8px;
+  scale: 85%;
+  transform: rotate(140deg) scaleX(-1);
+  transition: transform 0.2s ease-in-out, margin-left 0.2s ease-in-out;
+  width: 8px;
+}
+
+.graybox-container.open .gb-toggle::before {
+  margin-left: 3px;
+  transform: rotate(225deg) scaleX(1);
+}
+
+.graybox-container .graybox-menu {
+  background-color: var(--gb-container-bg);
+  border-radius: 3px;
+  border-top-right-radius: 0;
+  border: 0;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: 1fr 66px;
+  overflow: hidden;
+  position: absolute;
+  right: 39px;
+  width: 0;
+}
+
+.graybox-container .graybox-menu.hide-devices {
+  border-bottom-right-radius: 0;
+  grid-template-rows: 1fr 0;
+}
+
+.graybox-container.open .graybox-menu {
+  border: var(--gb-container-border);
+  min-height: 60px;
+  min-width: 320px;
+}
+
+.graybox-text {
+  color: #000;
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 24px;
+  padding: 16px;
+}
+
+.graybox-text > p:first-child {
+  font-weight: bold;
+  font-size: 24px;
+  margin: 0;
+}
+
+.graybox-devices {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  padding: 16px;
+  column-gap: 10px;
+  height: 34px;
+  border-top: var(--gb-container-border);
+  box-sizing: content-box;
+}
+
+.graybox-devices > a {
+  box-sizing: border-box;
+  width: 85px;
+  text-align: center;
+  color: var(--gb-container-btn-bg);
+  border-color: var(--gb-container-btn-bg);
+  cursor: default;
+}
+
+.graybox-devices > a:hover {
+  background-color: var(--gb-container-btn-bg);
+  text-decoration: none;
+}
+
+.dialog-modal.graybox-modal {
+  z-index: 2000;
+}
+
+.dialog-modal.graybox-modal.mobile > div {
+  overflow-x: hidden;
+  border-radius: 88px;
+}
+
+.dialog-modal.graybox-modal.tablet > div {
+  overflow-x: hidden;
+  border-radius: 88px;
+}
+
+.dialog-modal.graybox-modal.mobile .graybox-device-border {
+  position: absolute;
+  pointer-events: none;
+  width: 418px;
+  height: 863px;
+}
+
+.dialog-modal.graybox-modal.tablet .graybox-device-border {
+  position: absolute;
+  pointer-events: none;
+  width: 859px;
+  height: 1205px;
+}
+
+.dialog-modal.graybox-modal.mobile {
+  width: 417px;
```

Comment:
where do those exact px values comes from?

Reply:
Ah its the mobile emulator

Reply:
Fitted to "emulate" the iframe inside the mobile device frame.  Values were tweaked so that it fits exactly, including hiding the scroll bar.